spieleplattform-backend/src/main/kotlin/com/example/spieleplattformbackend/services/GameService.kt

```diff
@@ -0,0 +1,33 @@
+package com.example.spieleplattformbackend.services
+
+import com.example.spieleplattformbackend.models.Game
+import com.example.spieleplattformbackend.repositories.GameRepository
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+import java.util.*
+
+@Service
+class GameService(@Autowired val gameRepository: GameRepository) {
+    fun insertExampleGames() {
+        gameRepository.save(Game("Minecraft", Date(111, 10, 18), "Mojang Studios"))
+        gameRepository.save(Game("Die Sims 4", Date(114, 8, 2), "Maxis"))
+        gameRepository.save(Game("Grand Theft Auto V", Date(113, 8, 17), "Rockstar North"))
+        gameRepository.save(Game("Super Mario Bros.", Date(85, 8, 13), "Nintendo Research"))
+        gameRepository.save(Game("Tetris", Date(84, 5, 6), "Alexei Paschitnow"))
+    }
+
+    fun getGamesCount() : Long {
```

Comment:
In Kotlin, kannst du bei functions die nur ein Statement lang sind auch folgende shorthand syntax verwenden: `fun getGamesCount(): Long = gameRepository.countByIdNotNull()`